ansible/roles/spdashboard/tasks/main.yml

```diff
@@ -12,6 +12,14 @@
     - npm 
     - yarn
 
+- name: Install n in order to upgrade node
+  command: yarn global add n --prefix /usr/local
+  become: yes
```

Comment:
become: yes is not needed in the role. That has already been taken care of in the playbook

Reply:
My understanding is that adding become: yes runs the operation with super user privileges. But this is not the case? Or is the entire role run with sudo?

Reply:
The entire role is run with sudo, see https://github.com/SURFnet/sp-dashboard/blob/develop/ansible/vagrant.yml#L16

Reply:
:eyes: